ospo-mindmap/Content/ospomindmap.md

```diff
@@ -79,6 +79,7 @@ development team of the Linux Project
 
 ### 🔍 Oversee Open Source Compliance
 
+#### Compliance Considerations
```

Comment:
What other categories apart from "compliance considerations" can fit in the "oversee os compliance section"?

I just want to know if there are more sub-categories we can add apart from "compliance considerations" and list them here too.

Reply:
We have "Manage Open Source IT Infrastructure" that includes the "Compliance tooling", but in some organizations the OSPO is the one who is also running scans (SCA) and takes care of this kind of operations, so maybe that could be under "Oversee Open Source Compliance" too?

Reply:
I've just realized that you added "Compliance Considerations" to replace the missing title from the mind map 👍 

![Screenshot 2022-06-16 at 08-31-10 OSPO Mind Map](https://user-images.githubusercontent.com/43671777/174006291-a8aefeff-8447-4d0e-a835-6e572df6d817.png)

This segmentation works for me. Also, I prefer the idea of keeping the compliance tooling under the IT infrastructure. IMHO, A single team or OSPO role does no necessarily need to be addressed just on OSPO mindmap responsibility, some OSPOs might be performing several activities (e.g overseeing OS compliance and managing IT infrastructure)

Anyway, we can discuss it with the community in the upcoming AMER & APAC WDA or as a topic for TODO EU Sync and create a new PR if necessary 😉  

Thanks @winterrocks !

Reply:
Yep, I saw the lonely green node dot and figured that a heading would be nice for that and came up with "Compliance Considerations", which of course we should discuss and see if there are other and better names.